architecture/topics/nsxt.adoc

```diff
@@ -0,0 +1,11 @@
+The VMware NSX-T Data Center provides a policy-based overlay network reproducing the complete set of Layer 2
+through Layer 7 networking services (such as switching, routing, access control, fire-walling, and QoS) in software for
+native {product-title} networking capabilities.
+
+The NSX-T components can be installed and configured as part of the Ansible installation procedure which integrates an {product-title} SDN
+into a datacenter-wide NSX-T virtualised network connecting bare metal, virtual machines, and {product-title} pods.
+See the xref:../../install_config/configuring_nsxtsdn.adoc#install-config-configuring-nsxt-sdn[Installation] section for information on how to install and deploy OpenShift Container Platform with VMware NSX-T.
+
+The *NSX-T Container Plug-In* (NCP) integrates {product-title} into an NSX-T Data Center, which is typically configured for the entire datacenter. 
+
+For information on the NSX-T Data Cetner architecture and administration, see the link:https://docs.vmware.com/en/VMware-NSX-T-Data-Center/2.4/installation/GUID-10B1A61D-4DF2-481E-A93E-C694726393F9.html[VMware NSX-T Data Center v2.4 documentation] and the link:https://docs.vmware.com/en/VMware-NSX-T-Data-Center/2.4/com.vmware.nsxt.ncp_openshift.doc/GUID-1D75FE92-051C-4E30-8903-AF832E854AA7.html[NSX-T NCP configuration] guides.
```

Comment:
s/Cetner/Center 

Not sure Data Center is a proper noun here. If not, I think we can lower case it.

Reply:
Agree in this context it should probably say "NSX-T Manager" rather than "NSX-T Data Center"

Reply:
I believe that is the name of the product. 
https://www.vmware.com/products/nsx.html

Reply:
The *NSX-T Container Plug-In* (NCP) integrates {product-title} into an NSX-T Manager, which is typically configured for the entire datacenter.